main/build.gradle

```diff
@@ -335,6 +335,10 @@ dependencies {
     implementation "org.mapsforge:mapsforge-map-reader:$mapsforgeVersion"
     implementation "org.mapsforge:mapsforge-themes:$mapsforgeVersion"
 
+    // Settings Library + Appcompat for AndroidX
+    implementation 'androidx.appcompat:appcompat:1.2.0'
```

Comment:
AndroidX AppCompat is already specified below at line 383 - this isn't needed.

Reply:
Fixed via 423e660